docs/src/Box.doc.js

```diff
@@ -645,6 +645,70 @@ card(
     <Text color="white">"fit" sets width to 100% </Text>
   </Box>
 </Box>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`When content overflows the bounds of Box, there are multiple options to control the overflow behavior. The default is \`overflow="visible"\`, but the most common use case is supplying \`overflow="auto"\` to ensure overflow content can be accessed. Learn more about [CSS overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow).`}
+      title="Overflow"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function BoxFlyoutExample() {
```

Comment:
Formatted:


`function BoxFlyoutExample() {
  const longText =
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vitae nisl nec turpis vehicula ultrices. Duis pretium ut ipsum nec interdum. Vestibulum arcu dolor, consectetur ac eros a, varius commodo justo. Maecenas tincidunt neque elit, eu pretium arcu dictum ac. Donec vehicula mauris ut erat dictum, eget tempus elit luctus. In volutpat felis justo, et venenatis arcu viverra in. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Proin enim lorem, vulputate eget imperdiet nec, dapibus sed diam. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Suspendisse rhoncus ut leo non gravida. Nulla tincidunt tellus sit amet ornare venenatis. Sed quis lorem cursus, porttitor tellus sed, commodo ex. Praesent blandit pretium faucibus. Aenean orci tellus, vulputate id sapien sit amet, porta fermentum quam. Praesent sem risus, tristique sit amet pulvinar in, scelerisque sit amet massa.';

  return (
    <Flex gap={4} direction="row">
      <Flex gap={8} direction="column">
        <Box>
          <Text>Overflow Hidden</Text>
          <Box
            overflow="hidden"
            width={300}
            maxHeight={100}
            padding={2}
            borderStyle="sm"
            color="eggplant"
          >
            <Text color="white">{longText}</Text>
          </Box>
        </Box>
        <Box>
          <Text>Overflow Scroll</Text>
          <Box
            overflow="scroll"
            width={300}
            maxHeight={100}
            padding={2}
            borderStyle="sm"
            color="maroon"
            tabIndex={0}
          >
            <Text color="white">{longText}</Text>
          </Box>
        </Box>
        <Box>
          <Text>Overflow Visible</Text>
          <Box
            overflow="visible"
            width={300}
            maxHeight={100}
            padding={2}
            borderStyle="sm"
            color="purple"
          >
            <Text>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vitae nisl nec turpis
              vehicula ultrices. Duis pretium ut ipsum nec interdum. Vestibulum arcu dolor,
              consectetur ac eros a, varius commodo justo.
            </Text>
          </Box>
        </Box>
      </Flex>
      <Flex gap={8} direction="column">
        <Box>
          <Text>Overflow Auto</Text>
          <Box
            overflow="auto"
            width={300}
            maxHeight={100}
            padding={2}
            borderStyle="sm"
            color="eggplant"
          >
            <Box width={350} padding={2} borderStyle="sm" color="maroon" tabIndex={0}>
              <Text color="white">{longText}</Text>
            </Box>
          </Box>
        </Box>
        <Box>
          <Text>Overflow scrollX</Text>
          <Box
            overflow="scrollX"
            width={300}
            maxHeight={100}
            padding={2}
            borderStyle="sm"
            color="maroon"
          >
            <Box width={350} padding={2} borderStyle="sm" color="eggplant" tabIndex={0}>
              <Text color="white">{longText}</Text>
            </Box>
          </Box>
        </Box>
        <Box>
          <Text>Overflow scrollY</Text>
          <Box
            overflow="scrollY"
            width={300}
            maxHeight={100}
            padding={2}
            borderStyle="sm"
            color="maroon"
          >
            <Box width={350} padding={2} borderStyle="sm" color="eggplant" tabIndex={0}>
              <Text color="white">{longText}</Text>
            </Box>
          </Box>
        </Box>
      </Flex>
    </Flex>
  );
}`